utils/preprocess.py

```diff
@@ -49,9 +49,9 @@ def testBinary():
 
 def testDisplayImage():
     """ Function to test display image from .nii files. """
-    epi_img = nib.load('BrainHack-selected/LS600'+str(2)+'_T1_std.nii.gz')
+    epi_img = nib.load('../small_TestData/BrainHack-selected/LS600'+str(2)+'_T1_std.nii.gz')
     epi_img_data = epi_img.get_data()
-    epi_img1 = nib.load('BrainHack-selected/LS400'+str(5)+'_T1_std.nii.gz')
```

Comment:
Don't hardcode the specific files either. what if we want to use this with any general file that ends in `.nii.gz`